Rakefile

```
@@ -72,7 +72,7 @@ namespace :dependencies do
 
     task :install do
       fold("install.cocoapds") do
-        pod %w[install]
+        sh "bundle exec pod install || bundle exec pod install --repo-update"
```

🤔  The `Rakefile` is already run in the context of bundler (you're supposed to invoke it via `bundle exec rake dependencies` so not sure the call to `bundle exec` is necessary here… tho I see that the `pod` function that was used prior to this change does use `bundle exec` too… so meh, maybe we can keep it 🤷 

What do you think about still using that `pod` function helper that we've been using so far, and use ruby's `rescue` to catch when the first call failed and threw an exception, to try the repo-update then?

```suggestion
        pod %w[install] rescue pod %w[install --repo-update]
```